docs/examples/KModal/ModalWithSubmit&Cancel.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+
+  <div>
+    <KButton @click="openModal">Click Me</KButton>
+    <p v-if="showMessage">Successfully submitted the Form</p>
+    <KModal
+      v-if="showModal"
+      :appendToOverlay="true"
+      size="medium"
+      title="Submit Form"
```

Comment:
```suggestion
      title="Submit form"
```

Aligns case to writing guidelines.